gammapy/image/tests/test_mask.py

```diff
@@ -16,7 +16,11 @@ def test_random_creation():
     assert excluded[0].size != 0
 
 
-@pytest.mark.xfail
 def test_distance_image():
-    pass
-    # TODO
+    mask = ExclusionMask.empty(nxpix=300, nypix=200)
```

Comment:
As discussed in the GSOC call just now: it's not clear what the behaviour should be for input masks that are all or nothing excluded. @OlgaVorokh - Just add tests that establish the current behaviour for now, and then we discuss what we want.

I think maybe if nothing is excluded, the distance should be very large. We could put it to the max positive float.
And if everything is excluded, that's like being deep inside an exclusion regions. We could put the min negative float.
That behaviour should be most convenient when working with distance images in algorithms because it does the right thing (as opposed to having `NaN` and needing to special-case those cases in every algorithm.